packages/react-components/src/components/gif.tsx

```diff
@@ -116,7 +131,7 @@ const Gif = ({
         e.persist()
         setHovered(true)
         hoverTimeout.current = window.setTimeout(() => {
-            pingback.onGifHover(gif, user, e.target as HTMLElement)
+            pingback.onGifHover(gif, user, e.target as HTMLElement, attributes)
```

Comment:
I only added this to the React SDK, since that's the only place we'll probably actually need it